Packs/Hudsonrock/Integrations/Hudsonrock/Hudsonrock.yml

```diff
@@ -0,0 +1,106 @@
+commonfields:
+  id: Hudsonrock
+  version: -1
+name: Hudsonrock
+display: Hudsonrock
+category: Data Enrichment & Threat Intelligence
+description: "Enrichment from Hudsonrock free APIs at \n\nhttps://cavalier.hudsonrock.com/api/json/v2/osint-tools/\n\nSupports: IP, Email and Username query."
+configuration:
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Source Reliability
+  name: integrationReliability
+  defaultvalue: B - Usually reliable
```

Comment:
if this is fird party enrichment, probably it should be A+?

Reply:
I actually dropped it to D as default.